### PR TITLE
RTMP Live cuepoints

### PR DESCRIPTION
**Pull request by cladera**
Populate RTMP cuepoints to a in-band-metadata text track using WebVTT standard.

### CODE REVIEW

**Comment by cladera** [comment]
This feature depends on folks from [video-js-swf](https://github.com/videojs/video-js-swf) accept [my pull request](https://github.com/videojs/video-js-swf/pull/157).

File: dist/video-js/video.dev.js [hunk 1]
```diff
@@ -8135,6 +8135,26 @@ vjs.Flash['checkReady'] = function(tech){
 // Trigger events from the swf on the player
 vjs.Flash['onEvent'] = function(swfID, eventName){
   var player = vjs.el(swfID)['player'];
+  if(eventName == 'cuepoint'){
```
Comment (cladera) [inline dist/video-js/video.dev.js]: Not sure if this is the right place to do this

**Comment by cladera** [comment]
Just merged release 5.0.0 to this feature. 

Any comment about this request?

**Comment by forbesjo** [comment]
Hello could you open this PR against master and add some tests? Thanks!

**Pull request closed without merging**